pd/src/app.rs

```diff
@@ -281,30 +295,32 @@ impl App {
     /// so it is not safe to assume all checks performed in `CheckTx` were done.
     #[instrument(skip(self))]
     fn deliver_tx(&mut self, txbytes: Bytes) -> impl Future<Output = Result<Response, BoxError>> {
-        let finished_signal = self.completion_tracker.start();
         let state = self.state.clone();
-        let nct_root = self.note_commitment_tree.root2();
+        let recent_anchors = self.recent_anchors.clone();
```

Comment:
Similarly, why are we cloning the recent anchors here?

Reply:
cf https://github.com/penumbra-zone/penumbra/pull/180#discussion_r756450269 , the reason is the same so marking this arm as resolved and moving discussion there